apps/app/src/app/shared/layout/side-nav/side-nav-coming-soon.component.ts

```diff
@@ -1,9 +1,8 @@
 import { Component } from '@angular/core';
-import { HlmBadgeDirective } from '@spartan-ng/ui-badge-helm';
 
 @Component({
 	selector: 'spartan-side-nav-cs',
-	imports: [HlmBadgeDirective],
+	imports: [],
```

Comment:
super nit: Weird, looks like the migration command doesn't remove empty imports arrays. Should be an easy fix with find & replace, but idk if we need to be that particular about it - just something that caught my eye